packages/flame/lib/src/components/fps_component.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:collection';
+
+import '../../components.dart';
+
+/// The [FPSComponent] is a non-visual component which you can get the current
+/// fps of the game with by calling [fps], once the component has been added to
+/// the component tree.
+class FPSComponent extends Component {
+  FPSComponent({
+    this.windowSize = 60,
+  });
+
+  /// The number of game ticks over which the fps measure will be averaged.
+  final int windowSize;
```

Comment:
This name is a little bit confusing for me, at first I thought this had to do with the game window. Maybe we could call it `expectedFps` or something like that?

Reply:
I updated the dartdoc for the field